packages/vm/src/backend.rs

```diff
@@ -7,17 +7,10 @@ use cosmwasm_std::{Binary, ContractResult, SystemResult};
 #[cfg(feature = "iterator")]
 use cosmwasm_std::{Order, Record};
 
-<<<<<<< HEAD
 use crate::environment::Environment;
 use crate::{FunctionMetadata, WasmerVal};
 
-#[derive(Copy, Clone, Debug)]
-=======
-/// A structure that represents gas cost to be deducted from the remaining gas.
```

Comment:
why this comment is deleted?

Reply:
It seems that the comment was deleted by mistake.
Please check.

Reply:
LGTM!